processing/src/main/java/io/druid/query/dimension/LookupDimensionSpec.java

```diff
@@ -129,10 +129,10 @@ public ExtractionFn getExtractionFn()
     final LookupExtractor lookupExtractor = Strings.isNullOrEmpty(name)
                                             ? this.lookup
                                             : Preconditions.checkNotNull(
-                                                lookupReferencesManager.get(name),
-                                                "Lookup [%s] not found",
+                                                this.lookupReferencesManager.get(name).get(),
```

Comment:
Is this change needed in this PR? I don't really understand why the null check is taken out a level, and without tests it is not at all clear this change is accomplishing what it is supposed to.